index.html

```diff
@@ -2020,24 +1988,21 @@ <h3>Securing Mechanisms</h3>
 
         <p>
 This specification recognizes two classes of securing mechanisms: those that use
-external proofs and those that use embedded proofs. An
-<dfn class="export">external proof</dfn> is one that wraps an expression of
-this data model, such as via a JSON Web Token, which is elaborated on in the
-<em>Securing Verifiable Credentials using JOSE and COSE</em> [[VC-JOSE-COSE]] specification.
-An <dfn class="export">embedded proof</dfn> is a mechanism where the proof is
-included in the data model, such as a Data Integrity Proof, which is elaborated
-on in <em>Verifiable Credential Data Integrity</em> [[VC-DATA-INTEGRITY]].
-        </p>
-        <p>
-It should be noted that these two classes of securing mechanisms are not
-mutually exclusive.
-        </p>
-        <p>
-<a>Verifiable credentials</a> and <a>verifiable presentations</a> MUST be
-secured by at least one securing mechanism.
+enveloping proofs and those that use embedded proofs. An
+<dfn class="export">enveloping proof</dfn> is one that wraps a serialization of
+this data model. One such enveloping proof mechanism is JSON Web Token,
+which is elaborated on in the [[[VC-JOSE-COSE]]] [[VC-JOSE-COSE]]
```

Comment:
What does the syntax [[[VC-JOSE-COSE]]] [[VC-JOSE-COSE]] do?  That looks odd to me and it may be a mistake.

Reply:
It is not a mistake; it is a respec trick.

- [[[VC-JOSE-COSE]]] extracts the exact title of the publication directly into the text
- [[VC-JOSE-COSE]] creates a reference into the "references" section, and includes a formal document reference into the latter

Though useful, it is ugly... but that is beyond our control.